### PR TITLE
Mongo Database

### PR DESCRIPTION
**Pull request by Segelzwerg**
This adds the a mongo db integration. How ever a mongo db instance is needed on the host system.

- [ ] ~~Use [mongomock](https://github.com/mongomock/mongomock) or some other tool for creating in-memory db for testing.~~ moved to #59 
- [x] Documentation

### CODE REVIEW

**Comment by Segelzwerg** [comment]
> * [ ]  Use [mongomock](https://github.com/mongomock/mongomock) or some other tool for creating in-memory db for testing.

This is kinda done by running a mongodb service in the CI.

**Comment by Segelzwerg** [comment]
> > * [ ]   Use [mongomock](https://github.com/mongomock/mongomock) or some other tool for creating in-memory db for testing.
> 
> This is kinda done by running a mongodb service in the CI.

I created a separate issue for this.

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .pylintrc [hunk 1]
```diff
@@ -9,7 +9,7 @@ extension-pkg-allow-list=
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code. (This is an alternative name to extension-pkg-allow-list
 # for backward compatibility.)
-extension-pkg-whitelist=
+extension-pkg-whitelist=pydantic
```
[the remark this answers is not in the document]
Comment (Segelzwerg) [reply .pylintrc:12]: https://github.com/samuelcolvin/pydantic/issues/1961 No please look at last comment

[the remark this answers is not in the document]
Comment (iTitus) [reply .pylintrc:12]: -whitelist is deprecated, allow-list is the new alternative

[the remark this answers is not in the document]
Comment (Segelzwerg) [reply .pylintrc:12]: Ok I will change that

File: tests/whist/server/routes/test_index.py [hunk 2]
```diff
@@ -15,4 +15,4 @@ def test_read_index(self):
         """
         response = self.client.get('/')
         self.assertEqual(response.status_code, 200)
-        self.assertDictEqual(response.json(), {'game': 'whist'})
+        self.assertDictEqual(response.json(), {'info': {'game': 'whist', 'version': '0.1.0rc2'}})
```
Comment (iTitus) [inline tests/whist/server/routes/test_index.py:18]: can we not have that hardcoded in the test? so we dont have to change 10 different files to increment a version.

**Review by iTitus — CHANGES_REQUESTED** [review]

File: tests/whist/server/routes/test_index.py (hunk 2 above)
Comment (Segelzwerg) [reply tests/whist/server/routes/test_index.py:18]: Yes I was thinking about this. Will do this later

Comment (Segelzwerg) [reply tests/whist/server/routes/test_index.py]: The whist core version is not hard coded any more.

**Review by iTitus — APPROVED** [review]

**Pull request merged**